### PR TITLE
(See Desc) Dex Update

### PR DESCRIPTION
**Pull request by matteoruscitti**
Adds:
- Roedrunna
- Maliceros
- Breechon, Nimbichon
- Leviorca
- Squimshi, Gummelid
- Alivolt
- Fintricity
- Terrator
- Gengar Alola
- Intricat
(The latter two can be found exactly where their prevos are. Intricat evolves from Delcatty. I need a review on Gengar-Alola. There are still Pokemon missing. Trapinch-Orre line is missing, as is Vikavolt-Unova line. _In order for this update to be marked done, we must also complete tiering data and learnsets._ **Do not push this change until this is completed.**

**DEADLINE IS MAY 25, 2022**

### CODE REVIEW

File: data/pokedex.ts [hunk 1]
```diff
@@ -19937,4 +19967,124 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
...
+		color: "Black",
+		eggGroups: ["Water 2"],
+		newmeta: "true",
+	},
+	breechon: {
+		num: 10094,
+		name: "Breechon",
+		types: ["Flying"],
+		baseStats: {hp: 65, atk: 70, def: 40, spa: 47, spd: 30, spe: 70},
+		abilities: {0: "Cloud Nine", 1: "Fur Coat", H: "Aerilate"},
```
Comment (GamR-the-Great) [inline data/pokedex.ts:19999]: Breechon and Nimbichon are part of the same line, but you missed the evo & prevo stuff completely here.

File: data/pokedex.ts [hunk 2]
```diff
@@ -19937,4 +19967,124 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
...
+		baseStats: {hp: 80, atk: 70, def: 40, spa: 25, spd: 45, spe: 65},
+		abilities: {0: "Shield Dust", 1: "Sticky Hold", H: "Gluttony"},
+		heightm: 0.2,
+		weightkg: 5,
+		color: "Green",
+		eggGroups: ["Field", "Monster"],
+		newmeta: "true",
+	},
+	gummelid: {
+		num: 10098,
```
Comment (GamR-the-Great) [inline data/pokedex.ts:20043]: Same with Squimshi & Gummelid

**Review by GamR-the-Great — CHANGES_REQUESTED** [review]

**Comment by jmf61** [comment]
Added the tiering for Roedrunna through Terrator, still need tiering for Gengar-Alola and all mons listed below

**Comment by jmf61** [comment]
Added learnsets for Roedrunna through Terrator, still need learnsets for Gengar-Alola and all mons listed below

**Comment by matteoruscitti** [comment]
fintricity has "hydropum" rather than hydropump. Did the tool generate the miss spell or were these typed manually?

**Comment by jmf61** [comment]
> fintricity has "hydropum" rather than hydropump. Did the tool generate the miss spell or were these typed manually?

the submission had it misspelled, so autoformatting meant the code had same misspell

**Review by DomTom22 — APPROVED** [review]

**Pull request merged**